Userland/Applications/Spreadsheet/ImportDialog.cpp

```diff
@@ -176,7 +176,7 @@ void CSVImportDialogPage::update_preview()
 
 Result<NonnullRefPtrVector<Sheet>, String> ImportDialog::make_and_run_for(StringView mime, Core::File& file, Workbook& workbook)
 {
-    auto wizard = GUI::WizardDialog::construct(GUI::Application::the()->active_window());
+    auto wizard = GUI::WizardDialog::construct(GUI::Application::the()->active_window()->find_parent_window());
```

Comment:
This is a very rickety solution and will break if we change the way files are opened (which will happen once Spreadsheet is ported to FileSystemAccessClient)
Let's plumb the specific desired parent window here and use that instead of relying on the app's active window.

Reply:
Yeah, I thought of going that route but it felt weird because I would have had to have to change the the load method on the worksheet.

`Result<bool, String> Workbook::load(StringView filename)` to say 

`Result<bool, String> Workbook::load(Window* parent, StringView filename)`